src/muiTheme.tsx

```diff
@@ -110,6 +110,28 @@ const theme: any = createMuiTheme({
         color: 'white',
       },
     },
+    MuiTableSortLabel: {
+      root: {
+        color: white,
+        '&:hover': {
+          color: white,
+          opacity: 0.5,
+        },
+        '&$active': {
+          '&& $icon': {
```

Comment:
Is there a more material-ui way to do achieve this maybe? cc @Ry-DS

Reply:
I don't think so because it's how material-ui set the styles in `@material-ui/core/TableSortLabel/TableSortLabel.js`.

Reply:
I reckon there is actually. Looking at the applied classes: 
![image](https://user-images.githubusercontent.com/25571542/156766100-ad5288bc-ef5f-44f6-9f88-0fe21345ea13.png)
We should be able to target them directly with cleaner lables instead of this:
```ts
    MuiTableSortLabel: {
      root:{
        ...
      },
      active:{
        ...
      },
      icon:{
        ...
      }
    }
```

Reply:
@tdlinh2712 can you give this a shot?

Reply:
@ericboucher @Ry-DS doesn't work, I've tried using Ryan's suggestion, got:
![image](https://user-images.githubusercontent.com/30374401/156914111-db9bb069-effa-4237-9291-680c4fa2f53f.png)

, and
```
root: {
...
      },
      active: {
        '&& $icon': {
          opacity: 1,
          color: white,
        }
      },
```
, got:
![image](https://user-images.githubusercontent.com/30374401/156914054-6d82eef2-a90f-41ef-aa0a-0330e78b40a8.png)

Both were overwritten by the root class. I've tried it and other approaches over the past few days and I couldn't find one that worked apart from writing it inside the root element.

Reply:
Yup that's fair. I got it to work with `important` of course:
```
    MuiTableSortLabel: {
      root: {
        color: white,
        '&:hover': {
          color: white,
          opacity: 0.5,
        },
      },
      active: {
        '&>svg': { color: `${white} !important` },
      },
      icon: {
        color: white,
        opacity: 0.2,
        '&:hover': {
          opacity: 0.5,
        },
      },
    }
```
But honestly I reckon what we have here works fine (obvious to the reader that it activates on active)